### DISCUSSION TITLE
protect players from damage by zombies and slimes?

### DISCUSSION THREAD

**Discussion by RoyHinkley** [EssentialsX/Essentials#4322] [Q&A] [answered]
I'm a new user of EssentialsX, so perhaps I have missed something.

I see all sorts of options in the Protect plugin that prevent players from certain types of damage, but apparently not every possible sort of damage can be prevented (e.g., zombies and slimes).

I'm trying to prevent grey-listed players from getting hurt (and depleting hunger, different subject). I'd be grateful for any suggestions or ideas along these lines.

Cheers!

**Comment by JRoy** [comment, marked as answer]
Use WorldGuard.
